src/ecoscope-workflows-core/ecoscope_workflows_core/jsonschema.py

```diff
@@ -54,6 +54,9 @@ class oneOf(BaseModel):
     const: Any
     title: str
 
+    def __hash__(self) -> int:
+        return self.const.__hash__()
```

Comment:
Maybe something like

```suggestion
    def __hash__(self) -> int:
        return hash(("oneOf", self.const, self.title))
```

?

Reply:
Sure!